File: src/main/java/com/gmail/borlandlp/minigamesdtools/gun/BulletHandlerApi.java
package com.gmail.borlandlp.minigamesdtools.gun;

import com.gmail.borlandlp.minigamesdtools.APIComponent;
import com.gmail.borlandlp.minigamesdtools.gun.bullet.GhostBullet;

public interface BulletHandlerApi extends APIComponent {
    void addBullet(GhostBullet bullet);
    void removeBullet(GhostBullet bullet);
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gun/BulletHandlerApi.kt
package com.gmail.borlandlp.minigamesdtools.gun

import com.gmail.borlandlp.minigamesdtools.APIComponent
import com.gmail.borlandlp.minigamesdtools.gun.bullet.GhostBullet

interface BulletHandlerApi : APIComponent {
    fun addBullet(bullet: GhostBullet)
    fun removeBullet(bullet: GhostBullet)
}